projects/kubernetes-csi/external-resizer/1-29/CHECKSUMS

```diff
@@ -1,2 +1,2 @@
-b58e1add53aa84973fc6df2d46c3b4711ca8b777e0c6e8f0e95e5a164e588cbf  _output/1-29/bin/external-resizer/linux-amd64/csi-resizer
-8302bd5f6d96a17093b0162aac794fd56ab32d6ead1b06ddba3108b554117fc2  _output/1-29/bin/external-resizer/linux-arm64/csi-resizer
+37512c2c28fc239bbedf28489f4bf3fcd320565549675c71173c626ddbb3b825  _output/1-30/bin/external-resizer/linux-amd64/csi-resizer
```

Comment:
This says 1-30 when it should be 1-29 in checksums

EG

```
- 37512c2c28fc239bbedf28489f4bf3fcd320565549675c71173c626ddbb3b825 _output/1-30/bin/external-resizer/linux-amd64/csi-resizer
+ 37512c2c28fc239bbedf28489f4bf3fcd320565549675c71173c626ddbb3b825 _output/1-29/bin/external-resizer/linux-amd64/csi-resizer
```

I think similar thing is happening for other failures